<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepWidget</class>
 <widget class="QWidget" name="StepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>306</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="closeButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string comment="@info:tooltip">Close this tutorial</string>
       </property>
       <property name="whatsThis">
        <string comment="@info:whatsthis">Click here to close the tutorial.&lt;nl/&gt;The tutorial can be closed when you have finished it, or at any time to cancel it.</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ktutorial::view::StepTextWidget" name="textWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string comment="@info:whatsthis">Here appear the instructions for each step of the tutorial.&lt;nl/&gt;Once you complete one step, a new step with new instructions will be shown.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="optionsWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ktutorial::view::StepTextWidget</class>
   <extends>QTextEdit</extends>
   <header>StepTextWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
